geoprocessing/src/modules/shapefiles/shapefiles.service.ts

```diff
@@ -42,23 +24,14 @@ export class ShapeFileService {
     return JSON.parse(_geoJson[outputKey].toString('utf-8'));
   }
 
-  private deleteShapefileData(path: string): void {
-    if (path.startsWith('/tmp')) {
-      unlinkSync(path);
-      rmdirSync(path.replace('.zip', ''), { recursive: true });
-    } else {
-      throw new Error(`Could not complete deletion: ${path} is not in /tmp`);
-    }
-  }
-
   async getGeoJson(shapeFile: Express.Multer.File) {
```

Comment:
Would be great to see some unit tests with "simple" shapefile (both valid and invalid)